ravenframework/OutStreams/PlotInterfaces/GeneralPlot.py

```diff
@@ -620,10 +620,23 @@ def __executeActions(self):
       elif key == 'title':
         self.ax.set_title(self.options[key]['text'], **self.options[key].get('attributes', {}))
       elif key == 'scale':
+        major, minor = [int(x) for x in matplotlib.__version__.split('.')[:2]]
+        #matplotlib before 3.5 used nonpos instead of nonpositive
```

Comment:
nonposx and nonposy have been replaced with nonpositive in matplotlib 3.5, but have also been removed for anything other than a log scale.